### DISCUSSION TITLE
Resource Allocation / VM vs LXC

### DISCUSSION THREAD

**Discussion by fakebizprez** [henrygd/beszel#197] [Support]
Great work, I love the all-in-one solution.

Can you give a ballpark on the number of resources that should be dedicated to the host VM? Or could this even run on a Pocketbase LXC as a host?

**Comment by henrygd** [comment]
I think creating a VM just to host this would be overkill. It will run fine in a Linux Container.

It uses very little resources. Even one core should be okay. The most memory I've seen it use on my personal instance is 120 MB, with two users and about 15 agents hooked up. It uses 0.05% CPU on average on a 3 core Hetzner shared VPS.

The agents use either 0.00% or 0.01% CPU on my machines, and between 4 MB to 12 MB memory depending on the number of docker containers they need to track.

Let me know if I didn't understand you correctly. Not sure what you mean by Pocketbase LXC.
